src/test/java/hw2/Exercise1.java

```diff
@@ -0,0 +1,86 @@
+package hw2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+
+public class Exercise1 extends BaseTest{
+
+    @Test
+    public void testWebDriverSoftAsserts () {
+
+        SoftAssert softAssert = new SoftAssert();
+
+        //Open test site by URL
+        String openedURL = openSite();
+        softAssert.assertEquals(openedURL, URL);
+
+        //Assert Browser title
+        softAssert.assertTrue(homePageIsOpened());
+
+        //Perform login
+        softAssert.assertTrue(login());
+
+        //Assert Username is loggined
+        softAssert.assertTrue(checkUsername());
+
+        //Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> navigationBarButtons = driver.findElements(By.cssSelector("ul.uui-navigation > li"));
+        softAssert.assertEquals(navigationBarButtons.size(), 5);
+        for (WebElement element : navigationBarButtons){
+            softAssert.assertTrue(element.isDisplayed());
+        }
+        softAssert.assertEquals(navigationBarButtons.get(0).getText(), "HOME");
+        softAssert.assertEquals(navigationBarButtons.get(1).getText(), "CONTACT FORM");
+        softAssert.assertEquals(navigationBarButtons.get(2).getText(), "SERVICE");
+        softAssert.assertEquals(navigationBarButtons.get(3).getText(), "METALS & COLORS");
```

Comment:
This is a bit confused check. You verify that size of `navigationBarButtons` is 5, but in the next code you assert only 4 items in the `navigationBarButtons`